src/invidious/jsonify/api_v1/video_json.cr

```diff
@@ -245,6 +245,12 @@ module Invidious::JSONify::APIv1
                 json.field "lengthSeconds", rv["length_seconds"]?.try &.to_i
                 json.field "viewCountText", rv["short_view_count"]?
                 json.field "viewCount", rv["view_count"]?.try &.empty? ? nil : rv["view_count"].to_i64
+                json.field "published", rv["published"]?
+                if !rv["published"].nil?
```

Comment:
The '?' is required to prevent an error when loading previously cached video data after an update!
```suggestion
                if !rv["published"]?.nil?
```